src/lints/panic.rs

```diff
@@ -120,7 +121,8 @@ fn check_single_panic_usage(
                     stable_ptr: syntax_node.stable_ptr(db.upcast()),
                     message: PanicInCode.diagnostic_message().to_owned(),
                     severity: Severity::Warning,
-                    relative_span: None
+                    relative_span: None,
```

Comment:
FYI this will probably get reverted, since those fields are essentially the same functionality in the compiler :P